### PR TITLE
feat(network): Add ip filtering by cidr

### PR DESCRIPTION
**Pull request by yoniabrahamy**

### CODE REVIEW

**Comment by yoniabrahamy** [comment]
List of cidrs was acquired using API provided by https://networksdb.io/
https://github.com/networksdb/python

File: modules/processing/network.py [hunk 1]
```diff
@@ -98,12 +102,22 @@
...
 
-
 ip_passlist = set()
+network_passlist = []
+
 if enabled_ip_passlist and ip_passlist_file:
     with open(os.path.join(CUCKOO_ROOT, ip_passlist_file), "r") as f:
         ip_passlist = set(f.read().split("\n"))
 
+    if os.path.exists(network_passlist_file):
```
Comment (davidt99) [inline modules/processing/network.py]: isfile

File: modules/processing/network.py [hunk 2]
```diff
@@ -98,12 +102,22 @@
...
+        with open(os.path.join(CUCKOO_ROOT, network_passlist_file), "r") as f:
+            for cidr in list(set(f.read().splitlines())):
```
Comment (davidt99) [inline modules/processing/network.py]: no need for the list

File: modules/processing/network.py [hunk 3]
```diff
@@ -98,12 +102,22 @@
...
+                if cidr.startswith("#") or len(domain.strip()) == 0:
+                    # comment or empty line
+                    continue
+
+                network_passlist.append(ipaddress.ip_network(cidr))
```
Comment (davidt99) [inline modules/processing/network.py]: strip the cidr

**Review by davidt99 — APPROVED** [review]

**Pull request merged**